bread/layout/components/datatable.py

```diff
@@ -227,6 +226,19 @@ def with_toolbar(
                 ),
             )
 
+        searchbar = Search(
```

Comment:
I find it a bit confusing that we do all this work even if search_urlparameter is `None` and the 'searchbar' variable goes unused

Reply:
Right, refactored that.